### ISSUE TITLE
Warning on running program

### ISSUE THREAD

**Issue by LEGOlord208** [getlantern/systray#25]
`(yourprogramnamehere:987): Gdk-CRITICAL **: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed`

**Comment by mickael-kerjean** [comment]
Same here, did you manage to find a solution?

**Comment by jD91mZM2** [comment]
Sadly, nope. It's just a warning though so you should be able to ignore it

**Comment by mickael-kerjean** [comment]
Yes sure but from a end user perspective, trying to run a program that gives warning doesn't inspire confidence, I'll dig a bit more once in the next few weeks and will give some update if I have a solution

**Comment by Bios-Marcel** [comment]
Any new information on this?

**Comment by joesis** [comment]
One cause of this warning: https://git.xfce.org/apps/xfce4-terminal/commit/?id=eff5e2db3dff7e586193e95156da24f2acf33eeb

This package doesn't call the method directly but I guess app-indicator does that.

**Comment by Bios-Marcel** [comment]
Yeah, I have seen that issue aswel. I checked the code and couldn't find calls either. Anyways, if this is an libappindicator issue, should this issue be closed? If anyone can prove the opposite, the issue could be reopened.

**Comment by mickael-kerjean** [comment]
Said previously I'd investigate but haven't done much progress and decided to remove the feature instead of getting warning.

**Comment by Bios-Marcel** [comment]
I am a little confused. What does `the feature` mean in this context?

**Comment by mickael-kerjean** [comment]
`the feature` in the context of my project relying on this library. This was about having a way for users that aren't friend with the CLI to start a server/stop the server/launch the UI  for [this project](https://github.com/mickael-kerjean/nuage). 
At the end of the day, I don't have the manpower, devices and motivation to test this on all possible system and warnings always have this "euh ..." reaction when I launch a software that has some.
